charts/backstage/templates/servicemonitor.yaml

```
@@ -0,0 +1,32 @@
+{{- with .Values.serviceMonitor }}
+{{- if .enabled }}
```

I would prefer this to be:

```suggestion
{{- if .Values.serviceMonitor.enabled }}
```

---

I adopted this from the grafana chart and I kind of liked it. By changing the scope to the object in question, the rest of the value references, further down, requires less typing. I'll change to your preference, no bigge.